File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.java
import java.util.*;
class Solution {
    public String solution(int n) {
        String answer = "";
        
        StringBuilder sb = new StringBuilder();
        
        for(int i = 0; i < n; i++){
            sb.append(i % 2 == 0 ? "수" : "박");
        }
        
        answer = sb.toString();
        
        return answer;
    }
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int n) {
    char* answer = (char*)malloc((n * 3 + 1) * sizeof(char));
    
    answer[0] = '\0';
    
    for(int i = 1; i <= n; i++){
        if(i % 2 == 1){
            strcat(answer, "수");
        }else {
            strcat(answer, "박");
        }
    }
    
    return answer;
}